tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -167,6 +169,45 @@
 # It is quite clear that the MRI surfaces (head, brain) are not well aligned
 # to the head digitization points (dots).
 #
+# Compute the transform
+# ---------------------
+# Let's step through the process that the transform is accomplishing.
+
+###############################################################################
+# Get landmarks in head space from the DigMontage stored in raw.
+# -------------------------------------------------------------
+# Adjust units by * 1e3 m -> mm.
+# head_space = np.array([dig['r'] for dig in
+#                        raw.info['dig']], dtype=float) * 1e3
```

Comment:
You can do matplotlib 3D scatterplots for these and it would probably look okay

https://matplotlib.org/3.1.1/gallery/mplot3d/scatter3d.html